### DISCUSSION TITLE
status "paid" per una fattura

### DISCUSSION THREAD

**Discussion by zpj61** [fattureincloud/api#274] [Q&A] [answered]
Sto cercando di invirae una fattura pagata pertanto nei pagamenti inserisco lo status "paid" e l'id del conto di saldo come da json sottostante:
```
"payments_list": [
         {
            "due_date": "2023-02-01",
            "amount": 502,
            "status": "paid",
            "payment_terms": {
               "days": 1
            },
            "payment_account": {
               "id": 297038
            }
         }
      ]
```

ma la fattura la troviamo sempre con il pagamento "non saldato" sul web. C'è qualcosa di particolare da fare o da verificare?
Grazie

**Comment by emajo** [comment, marked as answer]
Ciao @zpj61 per contrassegnare una payment item come pagato è necessario passare anche il campo `paid_date`.

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiotno @zpj61 abbiamo risposto alla sua domanda?
